applications/2025/pending/subhopam-das-personal.yml

```
@@ -0,0 +1,113 @@
+# Ship-Every-Friday Squad Application
+# All we need is your GitHub - but tell us more if you want!
+
+essentials:
+  github_username: subhopam-das-personal 
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)